src/chipeitlib/src/main/java/es/chipeit/lib/extensions/Extensions.kt

```diff
@@ -2,3 +2,10 @@ package es.chipeit.lib.extensions
 
 internal val Byte.Companion.ZERO: Byte
```

Comment:
Let's nuke this one. It is only used in the `Timer` implementation to check if a byte is Zero. It is just useless to have a companion member just for that.

Reply:
Nuke the entire Extensions.kt file.